package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/HierarchyBuilder.cs

```diff
@@ -672,7 +672,7 @@ public struct Query<T> : IQuery where T : SampleBase, new() {
 
       if (!pl) {
         pl = go.AddComponent<UsdPayload>();
-        pl.SetInitialState(options.payloadPolicy == PayloadPolicy.LoadAll);
+        pl.SetInitialState(info.prim.GetPayloads().GetPrim().IsLoaded());
```

Comment:
You should be able to do `info.prim.IsLoaded() `

Reply:
Thanks! It is done in 67fc5e1.